CONTRIBUTING.md

```diff
@@ -157,22 +157,21 @@ import (
 )
 ```
 
-## Sign-off your work
+## Developer Certificate of Origin (DCO)
 
-The sign-off is a simple line at the end of the explanation for the
-patch. Your signature certifies that you wrote the patch or otherwise
-have the right to pass it on as an open-source patch. The rules are
-pretty simple: If you can certify [DCO](DCO), then you just add a line
-to every git commit message:
+We consider the act of contributing to the code by submitting a Pull
+Request as the "Sign off" or agreement to the certifications and terms
+of the [DCO](DCO) and [MIT license](LICENSE). No further action is required.
+Additionally you could add a line at the end of your commit message.
 
 ```
 Signed-off-by: Joe Smith <joe.smith@email.com>
 ```
 
-Please use your real name; we really dislike pseudonyms or anonymous
-contributions. We are in the open-source world without secrets. If you
-set your `user.name` and `user.email` git configs, you can sign-off your
-commit automatically with `git commit -s`.
+If you set your `user.name` and `user.email` git configs, you can add the
+line to the end of your commit automatically with `git commit -s`.
+
+We assume in good faith that the information you provide is legally biding.
```

Comment:
```suggestion
We assume in good faith that the information you provide adheres to the [DCO](DCO), and won't request more personal information than provided for contributions.
```

Not sure if this change makes sense with the DCO, but also we do allow pseudonyms and we probably aren't assuming they are 100% legally binding (and in many cases as the contributions aren't on a large scale to wonder if they were taken from anywhere else). Also I think with the information above most people who don't mind will do the sign off with legal name as they do now and people who don't want to won't